### PR TITLE
Create role & update from vsm1 and vsm2

### PR DESCRIPTION
**Pull request by nichenke**
This is a new role! We create the usual infrastructure for testing, then copy in the content from VSM1 vsm-config. After that, a few tweaks to get the role behaving better for us when using under Travis and Vagrant.

### CODE REVIEW

File: vagrant_variables.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+
+# Variables for in-repo testing with VirtualBox
+#
+
+# node parameters
+
+vm_memory: 1024
```
Comment (markswanusa) [inline vagrant_variables.yml:8]: Only a gig of mem?  What's going to be running on these nodes?

File: meta/main.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+galaxy_info:
+  author: Nic Henke <nic.henke@versity.com>
+  description: A really decent guy.
```
Comment (markswanusa) [inline meta/main.yml:3]: Pretty much the first thing I say when I mention you to friends.

**Review by markswanusa — CHANGES_REQUESTED** [review]

File: vagrant_variables.yml (hunk 1 above)
Comment (nichenke) [reply vagrant_variables.yml:8]: Just ansible to configure ssh...

File: meta/main.yml (hunk 2 above)
Comment (nichenke) [reply meta/main.yml:3]: I stole it from you.

File: vagrant_variables.yml (hunk 1 above)
Comment (markswanusa) [reply vagrant_variables.yml:8]: After slack conversation, I have a better understanding of the usage of this node.

**Review by markswanusa — APPROVED** [review]

**Pull request merged**